404.html

```diff
@@ -0,0 +1,195 @@
+                    <a href="https://twitter.com/404moldova" itemprop="sameAs"><i class="mdi-twitter mdi-set"></i></a>
+                    <a href="https://www.linkedin.com/company/404-moldova" itemprop="sameAs"><i class="mdi-linkedin mdi-set"></i></a>
+                    <a href="https://www.youtube.com/channel/UCXY-0ZUU-mvxbzWuLGZp5LQ" itemprop="sameAs"><i class="mdi-youtube-play mdi-set"></i></a>
+                    <a href="https://www.messenger.com/t/404Moldova/" itemprop="sameAs"><i class="mdi-facebook-messenger mdi-set"></i></a>
+                </div>
+            </div>
+            <div class="block-3 flex-item-4" itemscope itemtype="https://schema.org/PostalAddress">
+                <div class="right-side">
+                    <h4 class="connect-us">Get in touch</h4>
+                    <div class="one-line">
+                        <i class="mdi-email-outline mdi-set"></i>
+                        <a class="color-link" href="mailto:info@404.md" itemprop="email">info@404.md</a>
+                    </div>
+                    <div class="one-line">
+                        <i class="mdi-phone mdi-set"></i>
+                        <a class="color-link" href="tel:+37322808291">(+373) 22 808 291</a>
+                    </div>
+                    <div class="one-line">
+                        <i class="mdi-map-marker mdi-set"></i>
+                        <a class="color-link" itemprop="streetAddress" href="https://goo.gl/maps/GvA1ChgPJ4U2"><span itemprop="streetAddress">Blvd. Dacia #23, et. 3</span>
+                            <span class="one-line new-line" itemprop="addressLocality">Chisinau, Moldova</span></a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="footer-black">
+        <div class="container">
+            <div class="left-footer">
+                2017 &copy; PyCoding SRL
+                <a href="/terms/">Terms of Use</a>
+            </div>
+            <div class="right-footer">
+                <ul class="">
+                    <li><a href="/pricing/">Pricing</a></li>
+                    <li><a href="/members/">Members</a></li>
+                    <li><a href="/media/">Media</a></li>
+                    <li><a href="/gallery/">Gallery</a></li>
+                    <li><a href="/contact/">Contact</a></li>
+                    <li><a href="https://blog.404.md">Blog</a></li>
+                    <li><a href="/tour/">Book a Tour</a></li>
+                </ul>
+            </div>
+            <div class="clear"></div>
+        </div>
+    </div>
+</footer>
+
+<script type="text/javascript" src="/js/jquery-1.11.0.min.js"></script>
+<script type="text/javascript" src="/js/bodymovin.min.js"></script>
+<script type="text/javascript" src="/js/animations.js"></script>
+<script type="text/javascript" src="/js/main.min.js"></script>
```

Comment:
Let's get rid of `main.min.js` from 404.html (it can broke javascript code (we do not have `slick` on this page))